packages/components/src/local/map-bar/index.tsx

```diff
@@ -3,6 +3,10 @@ import cx from 'classnames'
 import { ArrowRight } from '@material-ui/icons'
 import assetDialogFor from './helpers/asset-dialog-for'
 
+import collatePlanFormData from './helpers/collate-plan-form-data'
+import collateAdjudicationFormData from './helpers/collate-adjudication-form-data'
+import collatePerceptionFormData from './helpers/collate-perception-form-data'
+
```

Comment:
Now we are pulling them in this way, it strikes me that those functions are very similar and could probably be refactored into a single function.

Reply:
Hmm, they share the `populate` and `values` general data structure, which I like. But the contents of those structures are quite different to each other.

If we change the structure of, say, the perception form, then we would expect `collatePerceptionFormData()` to be modified, but it would be more challenging to check for -ve consequences if we were modifying a single super-function, IMHO.

Reply:
Fair argument, I'm happy to leave it.